index.html

```diff
@@ -12,12 +12,12 @@
     <link href="https://fonts.googleapis.com/css2?family=Inter:wght@600&family=Roboto:wght@700&display=swap" rel="stylesheet">
 <body>
     <nav class="toolbar" id="toolbar">
-        <p class="logo">My Logo</p>
+        <a class="logo">My Logo</a>
```

Comment:
[OPTIONAL] You changed your logo from p to an element but let's add a href value so it can get clickable. Check this l[ink](https://www.w3schools.com/tags/att_a_href.asp) to learn more about a tag.